src/content/blog/cheerpj-3.0.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "CheerpJ 3.0"
```

Comment:
CheerpJ 3.0: A WebAssembly JVM to run real-world applications in the browser

It should fit in 80-chars, please fiddle as required